roles/huntarr/defaults/main.yml

```diff
@@ -0,0 +1,145 @@
+##########################################################################
+# Title:         Sandbox: Huntarr | Default Variables                    #
+# Author(s):     oryos                                                   #
+# URL:           https://github.com/saltyorg/Sandbox                     #
+# --                                                                     #
+##########################################################################
+#                   GNU General Public License v3.0                      #
+##########################################################################
+---
+################################
+# Basics
+################################
+
+huntarr_name: huntarr
+
+################################
+# Paths
+################################
+
+huntarr_paths_folder: "{{ huntarr_name }}"
+huntarr_paths_location: "{{ server_appdata_path }}/{{ huntarr_paths_folder }}"
+huntarr_paths_folders_list:
+  - "{{ huntarr_paths_location }}"
+
+################################
+# Web
+################################
+
+huntarr_web_subdomain: "{{ huntarr_name }}"
+huntarr_web_domain: "{{ user.domain }}"
+huntarr_web_port: "9705"
+huntarr_web_url:
+  "{{ 'https://' + (huntarr_web_subdomain + '.' + huntarr_web_domain
+  if (huntarr_web_subdomain | length > 0)
+  else huntarr_web_domain) }}"
+
+################################
+# DNS
+################################
+
+huntarr_dns_record: "{{ huntarr_web_subdomain }}"
+huntarr_dns_zone: "{{ huntarr_web_domain }}"
+huntarr_dns_proxy: "{{ dns.proxied }}"
+
+################################
+# Traefik
+################################
+
+huntarr_traefik_sso_middleware: ""
+huntarr_traefik_middleware_default: "{{ traefik_default_middleware }}"
+huntarr_traefik_middleware_custom: ""
+huntarr_traefik_certresolver: "{{ traefik_default_certresolver }}"
+huntarr_traefik_enabled: true
+huntarr_traefik_api_enabled: false
+huntarr_traefik_api_endpoint: ""
+
+################################
+# Docker
+################################
+
+# Container
+huntarr_docker_container: "{{ huntarr_name }}"
+
+# Image
+huntarr_docker_image_pull: true
+huntarr_docker_image_tag: "latest"
+huntarr_docker_image: "huntarr/huntarr:{{ huntarr_docker_image_tag }}"
+
+# Ports
+huntarr_docker_ports_defaults:
+  - "{{ huntarr_web_port }}:9705"
+huntarr_docker_ports_custom: []
+huntarr_docker_ports: "{{ huntarr_docker_ports_defaults
+  + huntarr_docker_ports_custom }}"
+
+# Envs
+huntarr_docker_envs_default:
+  TZ: "{{ tz }}"
+  PUID: "{{ uid }}"
+  PGID: "{{ gid }}"
```

Comment:
These are not a thing in this image. Runs as root always because the author has no clue how to do it properly based on commit history.